modules/apps/expando/expando-web/src/main/java/com/liferay/expando/web/internal/portlet/ExpandoPortlet.java

```diff
@@ -268,8 +238,18 @@ private Serializable _getDefaultValue(ActionRequest actionRequest, int type)
 		}
 
 		if (type == ExpandoColumnConstants.STRING_LOCALIZED) {
-			return (Serializable)LocalizationUtil.getLocalizationMap(
-				actionRequest, "defaultValueLocalized");
+			Serializable defaultValue =
+				(Serializable)LocalizationUtil.getLocalizationMap(
+					actionRequest, "defaultValueLocalized");
+
+			if (!_isDefaultLocaleValueInformed(
```

Comment:
@crodriguezy Could you try whether inline this method would make code hard to read? If not, please inline it.